Project/DAQ_System/Program/Src/Sensor/GyroScope/l3gd20h.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+* Gyroscope L3GD20H
+* Authors:       William Lim
+*                Alex Shi
+* Modified By:   Cristian Cruz
+*
+* Email:         wll180000@utdallas.edu
+*                ahs190001@utdallas.edu
+*                cris14.alex@gmail.com
+*
+* (c) 2023 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#include "l3gd20h.hpp"
+#include <stdint.h>
+
+
+namespace sensor{
+
+L3GD20H::L3GD20H(I2C_HandleTypeDef& hi2c):hi2c_(hi2c){
```

Comment:
Add a space. Change to `L3GD20H::L3GD20H(I2C_HandleTypeDef& hi2c):hi2c_(hi2c) {`